package.json

```diff
@@ -261,13 +261,15 @@
     "build-ext": "./node_modules/.bin/gulp buildExtension"
   },
   "devDependencies": {
```

Comment:
npm detected a vulnerability. Please run `npm audit fix` as part of this PR

Reply:
I pushed changes